main/modes/mode_tunernome.c

```diff
@@ -264,7 +264,7 @@ const uint16_t fourNoteStringIdxToLedIdx[] =
 
 const uint16_t fiveNoteStringIdxToLedIdx[] =
 {
-    1,
+    0,
```

Comment:
I thought using LEDs along the edge and ignoring the inner one was intentional, but if it looks that much better using the tighter ones...

Reply:
It was intentional, but in later testing when I forgot my rationale and came at it as a regular user, I thought there was a mistake or something because it looked like there was an empty spot for an LED between the first 2 and the 3rd on the column